### PR TITLE
Create IPFS-Browser-Protocol-Compliance-Suite.md

### PR DESCRIPTION
**Pull request by RangerMauve**
Hello,

Here's the targeted devgrant for creating an IPFS protocol compliance test suite.

CC @autonome @lidel

### CODE REVIEW

File: targeted-grants/IPFS-Browser-Protocol-Compliance-Suite.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+| 1 | Research/Description | $1300 | ~20h  |
+| 2 | Test Page with Compliance Report | $2600 | ~40h  |
+| 3 | Second page with POST/PUT | $1300 | ~20h  |
+| 4 | Passing coplinace in experimental browser | $1300 | ~20h  |
+
+## Acceptance Criteria
+
+<!-- What are the acceptance criteria for each milestone and for the final deliverables? These should be as objective as possible. They will be used to determine whether or not a grantee will receive payment for work completed for a milestone. -->
+
+TODO
```
[the remark this answers is not in the document]
Comment (RangerMauve) [reply targeted-grants/IPFS-Browser-Protocol-Compliance-Suite.md:80]: These look perfect, thank you!

**Comment by RangerMauve** [comment]
Committed proposed changes. This all LGTM! 😁

**Review by lidel — APPROVED** [review]
Thank you, lgtm.

@autonome mind taking the final look? What is the next step on the formal side here?

**Comment by autonome** [comment]
Thanks for working on this grant! Please create a discussion thread in the format of `[GRANT] Your Grant Title` in the community repo and provide updates on your milestones on a weekly basis: https://github.com/ipfs/community/discussions

**Comment by RangerMauve** [comment]
Thank you! I'm going to get started either end of the week or beginning of next week. 😁

**Comment by RangerMauve** [comment]
Discussion page has been created here: https://github.com/ipfs/community/discussions/573

Getting started this week. 😁

**Pull request merged**